README.md

```diff
@@ -33,7 +33,7 @@ mist:change-acl
 
 ## Building
 
-To build the bundle, run the following:
+To build the bundle you will need python, pip and PyYAML installed. Then just run the following:
```

Comment:
pycog isn't necessarily installed at this point @christophermaier. It only get's installed when the command is installed on the relay. We need yaml parsing for validation when building the cog file.